README.md

```diff
@@ -1,4 +1,29 @@
 ![alt text](https://travis-ci.org/vkorobkov/jfixtures-cmd.svg?branch=master "Build status")
 
 # jfixtures-cmd
-Command line interface for Jfixtures project
+Command line interface for JFixtures project
+
+## Dependencies
+**JFixtures requires Java 8** or higher and it is written in Java 8.
+
+## Usage
+* Build an executable jar with required dependencies by executing `mvn clean install`
```

Comment:
Let's remove this point for now - when we publish the artifact to maven central we'll provide a direct link for downloading the JAR file